WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/InitializationManager.java

```diff
@@ -60,6 +63,9 @@ private InitializationManager() {
 				new IntervalTransferManager(intervalsToTransferPersister,
 						WatchDogUtils.getWorkspaceName()));
 		workbenchListener.attachListeners();
+
+		DebugPlugin.getDefault()
```

Comment:
Wouldn't this call make more sense within workbenchListener.attachListeners(); ?